sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/utils/storage/DatabaseManagerAndroidTest.java

```diff
@@ -84,6 +85,8 @@ public static void tearDownClass() {
         sContext.deleteDatabase("test-replace-by-multiple-columns");
         sContext.deleteDatabase("test-multiple-tables");
         sContext.deleteDatabase("test-multiple-tables-read-write");
+        sContext.deleteDatabase("replace-with-uniqueness-constraint");
```

Comment:
I see this list of databases to tear down and I worry that it's not going to be scalable when we keep adding tests (speed, storage, etc.)  Is there some reason why we don't add setup/teardown methods that run after every test and reuse one of these databases?

Reply:
Great point @jwallra. Addressed in https://github.com/Microsoft/AppCenter-SDK-Android/pull/1061/commits/5b1c9d490a6543e091dd62e7f08984c0c43302c5#diff-5729da4fb091f62487cfcc9e97edb69f.